File: Platazi_Learn/POO/PHP/car.php
<?php

class Car {
public $id;
public $license;
public $driver;
protected $passengers;

function __construct($license, $driver) {
    $this->license = $license;
    $this->driver = $driver;
}

public function getPassengers() { 
    return $this->passengers;
}

public function setPassengers($passengers) {
    if ($passengers > 0 && $passengers <= 4) {
        $this->passengers = $passengers;
    } else {
        echo "\nEl numero de pasajeros en este auto es de uno (1) a cuatro (4) pasajeros por viaje\n";
    }
}

public function PrintDataCar() {
    if ($this->passengers != null) {
        echo "
        Licencia: $this->license 
        Driver: {$this->driver->name} 
        Número de pasajeros: $this->passengers
    ";
        }
    }
}

File: Platazi_Learn/POO/Java/Car.java
package Java;

public class Car {
    Integer id;
    private String license;
    private Account driver;
    protected Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    public Integer getPassenger(){ 
            return passenger;
    }

    void setPassenger(Integer passenger){
        if (passenger > 0 && passenger <= 4) {
            this.passenger = passenger;
        } else {
            System.out.println("\n");
            System.out.println("Solo pueden viajar de 1 hasta 4 pasajeros maximo, en este vehiculo");
        }

    }
    
    void printDataCar() {    
        if (passenger != null) {
            System.out.println("\n");
            System.out.println("License: " + license);
            System.out.println("Driver: " + driver.name);
            System.out.println("Passenger: " + passenger);
        }
    }
}
